x-pack/legacy/plugins/ingest/server/libs/types.ts

```diff
@@ -50,6 +50,8 @@ export interface Datasource extends SavedObjectAttributes {
   package: Package;
   read_alias?: string;
   streams: Stream[];
+  // TODO: These are the input parts of the streams. To be remove again when we have the streams
+  inputs: string[];
```

Comment:
@nchaulet My temporary hack to get the inputs into the datasource. This is an array of input configs. Each must be completed with the output to become a full stream.

Reply:
@ruflin  what about assigning `output_id: 'default', as we only support default output for now so we can have streams directly?